lib/PrincipleStudios.OpenApi.CSharp/Templates/objectmodel.handlebars

```diff
@@ -14,6 +14,11 @@ namespace {{this.PackageName}}
         [property: global::System.Text.Json.Serialization.JsonPropertyName("{{BaseName}}")]{{#if Required
             }}[property: global::System.Text.Json.Serialization.JsonRequiredAttribute]{{else
             }}[property: global::System.Text.Json.Serialization.JsonIgnore(Condition = global::System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingDefault)]{{/if}}{{!
+            }}{{#if Required}}[global::System.ComponentModel.DataAnnotations.Required]{{/if}}{{!
```

Comment:
@mdekrey This Required attribute is breaking some of the tests, specifically around the the 'nullable' properties marked as required in the schema (at least from what I can tell). ModelState is invalid.  Any advice on this? Thanks.

Reply:
.NET's `RequiredAttribute` has slightly different meanings than OpenAPI's `required` array - empty strings are considered missing to .NET. Let me take a look.